packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```
@@ -25,6 +25,8 @@ BuildArch: noarch
 Requires: (rubygem(base64) or ruby-default-gems < 3.4)
 Requires: (rubygem(csv) or ruby-default-gems < 3.4)
 
+Provides: hammer-cli = %{version}%{?prerelease}
```

Do we need to include the release here too?
```suggestion
Provides: hammer-cli = %{version}%{?prerelease}-%{release}
```